src/metabase/util/time/impl.cljs

```diff
@@ -356,19 +356,19 @@
 
 (def ^:private temporal-formats
   {:offset-date-time {:regex   common/offset-datetime-regex
-                      :formats #js ["yyyy-MM-DDTHH:mm:ss.SSS[Z]"
```

Comment:
`y` in moment means `era` and `Y` means year; it seems that the function wasn't tested (even indirectly) before